package/gluon-iptables-clamp-mss-to-pmtu/files/lib/gluon/mesh-vpn/iptables-mss.rules

```diff
@@ -1,3 +1,3 @@
 *mangle
--A FORWARD -o mesh-vpn -p tcp -m tcp --tcp-flags SYN,RST SYN -d 64:ff9b::/96 -j TCPMSS --clamp-mss-to-pmtu
+-A FORWARD -o mesh+ -p tcp -m tcp --tcp-flags SYN,RST SYN -j TCPMSS --clamp-mss-to-pmtu
```

Comment:
Why clamping on the wireless mesh interfaces? Their MTU is 1500.

Reply:
this is meant for all mesh interfaces keeping the same logic for fastd and wireguard.
As you observed this includes the wireless interfaces. I would argue that there is not much harm done in doing that.

Reply:
Except that the kernel has to process the clamping rule for every package going over the interface, I'm not sure that is wise.

Reply:
ok. I could rename the vpn devices as a workaround to mesh-vpn-wg-blaaa and clamp on mesh-vpn+ instead.